sdcard/RES/DO NOT MANUALLY EDIT FILES HERE.txt

```diff
@@ -0,0 +1,14 @@
+This folder contains the resources files that
+you should overwrite during sdcard resources updating.
+
+You can of course edit files/dirs here,
+however it's not necessary at all.
+
+You can add your own
+lists / replay files / playlist files
+in the /USR/ dir of your sdcard.
+
+Firmware will settle your files
+and system files automatically;
+so you won't need to combime and merge
+them together every update.
```

Comment:
I'd propose:

```
This folder contains resource files that
may be overwritten when updating the 
SD Card contents.

You can of course edit files/folders
here, but it's recommended that you
place your own files in the /USR
folder.

Firmware apps will check in both
/USR and /RES folders for files.
```

Reply:
Thank you very much! 
Suggestion accepted.